glotaran/model/model.py

```diff
@@ -34,12 +34,15 @@
 }
 
 default_dataset_properties = {
+    "clp_guidance": {"type": str, "allow_none": True},
+    "overwrite_model_dimension": {"type": str, "allow_none": True},
+    "overwrite_global_dimension": {"type": str, "allow_none": True},
```

Comment:
override

Reply:
Fixed in [7f565aa](https://github.com/glotaran/pyglotaran/pull/983/commits/7f565aac69c6433bfdc04444c3b2f0dd1997b31f)